src/main/java/agh/ics/oop/OptionsParser.java

```diff
@@ -0,0 +1,42 @@
+package agh.ics.oop;
+
+import java.util.Objects;
+
+public class OptionsParser {
+    public MoveDirection[] parse(String [] args)
+    {
+
+        int dir_len = 0;
+        for (int i=0;i<args.length;i++) {
+            if (Objects.equals(args[i], "f") || Objects.equals(args[i], "foward") || Objects.equals(args[i], "b") || Objects.equals(args[i], "backward") || Objects.equals(args[i], "l") || Objects.equals(args[i], "left") || Objects.equals(args[i], "r") || Objects.equals(args[i], "right"))
+                dir_len += 1;
+            else
+                args[i] = "";
```

Comment:
abstrahujac od rozwiązania to niszczenie wejściowej tablicy chyba zupełnie zbędne? generalnie unikajmy niszczenia parametrów wejściowych na potrzeby lokalnego algorytmu